flutter/cpp/datasets/README.md

```diff
@@ -126,3 +126,68 @@ you need to replace `squad_eval_mini.tfrecord` by `squad_eval.tfrecord` in the
       --ground_truth_directory=/tmp/ade20k_512/annotations/raw  \
       --num_threads=4
     ```
+
+## SNU SR set
+
+preparing datasets and models
```

Comment:
You have all the code and how to prepare the dataset and the models in the linked issue 👍. Would it make sense to also include those in this PR?